android/src/main/java/com/amplitude/android/plugins/AndroidContextPlugin.kt

```diff
@@ -150,6 +150,11 @@ class AndroidContextPlugin : Plugin {
                 event.plan = it
             }
         }
```

Comment:
Nit: maybe we should copy the `amplitude.configuration.ingestionMetadata` to prevent unintended modification by enrichment plugins?